_security/configuration/disable-enable-security.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: default
+title: Disabling and enabling the Security plugin
+parent: Configuration
+nav_order: 40
+has_toc: true
+redirect_from: 
+ - /security-plugin/configuration/disable/
+---
+
+# Disabling and enabling the Security plugin
+
+The Security plugin is installed by default with OpenSearch, but you can disable it temporarily if you want to make testing or internal usage more straightforward. You can then enable it once you're ready to configure security for your cluster. 
```

Comment:
Nit: This line reads a little weird to me. I would consider rewording "if you want to make testing or internal usage more straightforward".

For instance "but you can temporarily disable it to streamline testing."

Reply:
change added